per/views.py

```diff
@@ -429,6 +429,34 @@ def handle_post(self, request, *args, **kwargs):
 
         return JsonResponse({'status': 'ok'})
 
+def delete_draft(raw):
+    Draft.objects.filter(id=raw['id']).delete()
+    return
+
+class DelDraft(PublicJsonPostView):
+    def handle_post(self, request, *args, **kwargs):
+        u = None
```

Comment:
Hmm is it not possible to get a reference to the user directly with `request.user` ? Having to get the token from the headers and manually look this up seems like it should not be required - i.e. the framework should handle this. Let me experiment with this a bit and maybe suggest a better method to get the user details from the request.